actors.md

```diff
@@ -477,31 +461,53 @@ type StorageMinerActorState struct {
     dePledgeTime BlockHeight
 
 	## All sectors this miner has committed.
-    sectors SectorSet
+    sectors &SectorSet
 
 	## Sectors this miner is currently mining. It is only updated
 	## when a PoSt is submitted (not as each new sector commitment is added).
-    provingSet SectorSet
+    provingSet &SectorSet
```

Comment:
In this issue (https://github.com/filecoin-project/specs/issues/346), we talked about provingSet being a BitField, should we update that here?